docs/patterns/actions/action-guide-with-agents.mdx

```diff
@@ -68,4 +68,4 @@ agent_executor.invoke({"input": task})
 </CodeGroup>
 
 This code demonstrates how to use LLMs to execute an action. 
-Composio supports multiple frameworks for creating Agents including LlamaIndex, CrewAI, Letta, Langchain, you can see it in the **Supported Frameworks** section.
+Composio supports multiple frameworks for creating Agents including LlamaIndex, CrewAI, Letta, LangChain, you can see it in the **Supported Frameworks** section.
```

Comment:
Good correction of "Langchain" to "LangChain". Consider adding brief descriptions of each mentioned framework (LlamaIndex, CrewAI, Letta, LangChain) to provide more context for users who might be unfamiliar with these tools.

Reply:
this section is related to actions & there are independent section explaining each famework so didn't repeat it again here.